cmd/controller-manager/main.go

```diff
@@ -16,6 +16,7 @@ package main
 import (
 	"context"
 	"flag"
+	"github.com/pingcap/tidb-operator/pkg/controller/tidbmonitor"
```

Comment:
move this package to the second part with prefix github.com

Reply:
updated.